test-suite/src/test/scala/org/scalajs/testsuite/javalib/LinkedListTest.scala

```diff
@@ -0,0 +1,226 @@
+
+  override def testApi(): Unit = {
+
+    super.testApi()
+
+    it("could be instantiated with a prepopulated Collection") {
+      val s = Seq(1, 5, 2, 3, 4)
+      val l = asJavaCollection(s)
+      val ll = new LinkedList[Int](l)
+
+      expect(ll.size()).toEqual(5)
+
+      for (i <- 0 until s.size)
+        expect(ll.poll()).toEqual(s(i))
+
+      expect(ll.isEmpty).toBeTruthy
+    }
+
+    it("should add multiple element in one operation") {
+      val l = asJavaCollection(Set(1, 5, 2, 3, 4))
+      val ll = new LinkedList[Int]()
+
+      expect(ll.size()).toEqual(0)
+      ll.addAll(l)
+      expect(ll.size()).toEqual(5)
+      ll.add(6)
+      expect(ll.size()).toEqual(6)
+    }
+
+    it("could be instantiated with a prepopulated Collection") {
+      val s = Seq(1, 5, 2, 3, 4)
+      val l = asJavaCollection(s)
+      val ll = new LinkedList[Int](l)
+
+      expect(ll.size()).toEqual(5)
+
+      for (i <- 0 until s.size)
+        expect(ll.poll()).toEqual(s(i))
+
+      expect(ll.isEmpty).toBeTruthy
+    }
+
+    it("should retrieve the last element") {
+      val llInt = new LinkedList[Int]()
+
+      expect(llInt.add(1000)).toBeTruthy
+      expect(llInt.add(10)).toBeTruthy
+      expect(llInt.pollLast()).toEqual(10)
+
+      val llString = new LinkedList[String]()
+
+      expect(llString.add("pluto")).toBeTruthy
+      expect(llString.add("pippo")).toBeTruthy
+      expect(llString.pollLast()).toEqual("pippo")
+
+      val llDouble = new LinkedList[Double]()
+
+      expect(llDouble.add(+10000.987)).toBeTruthy
+      expect(llDouble.add(-0.987)).toBeTruthy
+      expect(llDouble.pollLast()).toEqual(-0.987)
+    }
+
+    it("should perform as a stack with push and pop") {
+      val llInt = new LinkedList[Int]()
+
+      llInt.push(1000)
+      llInt.push(10)
+      expect(llInt.pop()).toEqual(10)
+      expect(llInt.pop()).toEqual(1000)
+      expect(llInt.isEmpty()).toBeTruthy
+
+      val llString = new LinkedList[String]()
+
+      llString.push("pluto")
+      llString.push("pippo")
+      expect(llString.pop()).toEqual("pippo")
+      expect(llString.pop()).toEqual("pluto")
+      expect(llString.isEmpty()).toBeTruthy
+
+      val llDouble = new LinkedList[Double]()
+
+      llDouble.push(+10000.987)
+      llDouble.push(-0.987)
+      expect(llDouble.pop()).toEqual(-0.987)
+      expect(llDouble.pop()).toEqual(+10000.987)
+      expect(llString.isEmpty()).toBeTruthy
+    }
+
+    it("should find index of provided elements") {
+      val l = asJavaCollection(Seq("one", "two", "three", "two", "one"))
+      val ll = new LinkedList[String](l)
+
+      expect(ll.indexOf("one")).toEqual(0)
+      expect(ll.lastIndexOf("one")).toEqual(4)
+      expect(ll.indexOf("two")).toEqual(1)
+      expect(ll.lastIndexOf("two")).toEqual(3)
+      expect(ll.indexOf("three")).toEqual(2)
+      expect(ll.lastIndexOf("three")).toEqual(2)
+    }
```

Comment:
This test is redundant with
https://github.com/scala-js/scala-js/blob/8ce9654b3b3d6493113ad090fcc2093660ee0a7f/test-suite/src/test/scala/org/scalajs/testsuite/javalib/ListTest.scala#L242
and
https://github.com/scala-js/scala-js/blob/8ce9654b3b3d6493113ad090fcc2093660ee0a7f/test-suite/src/test/scala/org/scalajs/testsuite/javalib/ListTest.scala#L257